dev-requirements.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
requirements.txt is dev-requirements, add the rest into here and delete the old file